From 9639b0f06c70bb94b9112dcb3845fad8819e4802 Mon Sep 17 00:00:00 2001
From: Basil Chew <basil.chew@intel.com>
Date: Mon, 30 Jan 2023 11:09:23 +0800
Subject: [PATCH 17/18] Update SRIOV QEMU patching

Changes:
- added patching for SRIOV QEMU CVE patches
- updated SRIOV QEMU patch path

Change-Id: Iae178053579f93b9a15c034e37946ef78481c9a4

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index c21e0f4..88412e8 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -71,9 +71,17 @@ function ubu_install_qemu_gvt(){
         for i in $CIV_GOP_DIR/qemu/*.patch; do patch -p1 < $i; done
     fi
 
-    sriov_qemu_patch_num=$(ls $CIV_WORK_DIR/sriov_patches/qemu/*.patch 2> /dev/null | wc -l)
+    sriov_qemu_cve_patch_num=$(ls $CIV_WORK_DIR/sriov_patches/qemu/cve/*.patch 2> /dev/null | wc -l)
+    if [ "$sriov_qemu_cve_patch_num" != "0" ]; then
+        for i in $CIV_WORK_DIR/sriov_patches/qemu/cve/*.patch; do
+            echo "applying qemu patch $i"
+            patch -p1 < $i
+        done
+    fi
+
+    sriov_qemu_patch_num=$(ls $CIV_WORK_DIR/sriov_patches/qemu/qemu/*.patch 2> /dev/null | wc -l)
     if [ "$sriov_qemu_patch_num" != "0" ]; then
-        for i in $CIV_WORK_DIR/sriov_patches/qemu/*.patch; do
+        for i in $CIV_WORK_DIR/sriov_patches/qemu/qemu/*.patch; do
             echo "applying qemu patch $i"
             patch -p1 < $i
         done
-- 
2.40.0

